### DISCUSSION TITLE
No EIP-6963 Provider Detected in E2E Test Dapp

### DISCUSSION THREAD

**Discussion by philpetrov** [Synthetixio/synpress#1036] [General]
Hi Guys! 

I am using cypress version 12.17.3 and synpress version @synthetixio/synpress": "^3.7.1, when I went to the site https://metamask.github.io/test-dapp/ in the autotest I received No EIP-6963 Provider Detected and I can’t connect metamask, it worked before, how can I solve this? Thank you!

![image](https://github.com/Synthetixio/synpress/assets/63842606/613db0e1-80b2-42b9-9beb-a1c423be25df)

**Comment by duckception** [comment]
Synpress uses an older version of MetaMask that doesn't support this EIP. In order to get around this, you'll have to host your own MetaMask test dapp locally: https://github.com/MetaMask/test-dapp. Try version `v7.0.0`.
